.gitignore

```diff
@@ -3,6 +3,8 @@
 !jest.config.js
 *.d.ts
 node_modules
+resources/jenkins.yaml
```

Comment:
Can you add a comment here explaining why are we ignoring this file?

Reply:
will do

Reply:
thanks

Reply:
Added